.gitignore

```diff
@@ -11,10 +11,20 @@
 /*tags
 /autom4te.cache
 
-# /build/
+# Autoconf/Bakefile
+Makefile
+*.o
+*.so
+*.so.*
+.deps
+aclocal.m4
+bk-deps
+lib/wx/
+utils/ifacecheck/rungccxml.sh
+utils/wxrc/wxrc
+version-script
```

Comment:
Sorry, no, we don't want to have this here. Please run the builds in a different directory.

Reply:
Okay. I'll drop this commit.